static/app/types/index.tsx

```diff
@@ -383,8 +389,8 @@ export type EventMetadata = {
   origin?: string;
   function?: string;
   stripped_crash?: boolean;
-  current_tree_label?: string[];
-  finest_tree_label?: string[];
+  current_tree_label?: TreeLabelComponent[];
```

Comment:
should we name it only `TreeLabel`?

Reply:
renamed to treelabelpart for consistency with backend

the array is the label